src/client/v2/algod/models/types.ts

```diff
@@ -440,11 +440,15 @@ export class AccountApplicationResponse extends BaseModel {
    * global data.
    * The raw account uses `AppParams` for this type.
    */
-  constructor(
-    round: number | bigint,
-    appLocalState?: ApplicationLocalState,
-    createdApp?: ApplicationParams
-  ) {
+  constructor({
```

Comment:
@algochoi Unrelated to the line and minor:  Can you supply a CHANGELOG update to help the reader understand how to migrate?

Reply:
Thanks for the reminder, I keep forgetting this. I'll supply a commit here.

Reply:
[9b312db](https://github.com/algorand/js-algorand-sdk/pull/720/commits/9b312db44ccaff7a62d0ce8ef8180fe67da87b28)

Reply:
@algochoi Resolving - Thanks for bringing in the CHANGELOG.